selenium-webdriver/http.d.ts

```diff
@@ -100,10 +100,11 @@ declare namespace http {
      */
     class Executor {
         /**
-         * @param {!HttpClient} client The client to use for sending requests to the
-         *     server.
+         * @param {!(HttpClient|IThenable<!HttpClient>)} client The client to use for sending
```

Comment:
https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/http/index.js#L252 It looks like `class Executor` is still exported there, though the actual definition is in another file.